static/js/client.js

```diff
@@ -416,3 +416,23 @@ socket.on("waittimes", function(message) {
     waittimes = message.times;
     updateStatus();
 });
+
+socket.on("notifytime", function(message) {
+    if (disable_updates) return;
+    checkAndUpdateSeq(message.seq);
```

Comment:
If we don't increment seq on the server, we don't need to check and update here.